elliottlib/cli/advisory_drop_cli.py

```diff
@@ -44,7 +48,8 @@ def advisory_drop_cli(advisory):
     data = 'utf8=%E2%9C%93&reason=Dropping+unused+advisory%21&commit=Dropping+unused+advisory'
     headers = {"Content-Type": "text/plain"}
     r = requests.post(url, auth=HTTPSPNEGOAuth(), data=data, headers=headers)
-    if not r.ok:
+    adv = Erratum(errata_id=advisory)
+    if adv.errata_state != "DROPPED_NO_SHIP":
```

Comment:
I don't like refetching here, we should trust the return code. I would like to print the return error output if we get any

Reply:
the return code is not reliable, see https://saml.buildvm.hosts.prod.psi.bos.redhat.com:8888/job/hack/job/ximhan/job/build%252Fdrop_advisories/3/console it returns 500 but actually advisory already dropped, we don't need to check the return code as long as the advisory status is dropped